problems

```diff
@@ -0,0 +1,66 @@
+<?php
+//problema1 
+function sumarray($arr){
+    $sum=0;
+	for($i=0;$i<count($arr);$i++)
+	{
+		$sum=$sum+$arr[$i];
+	}
+    return $sum;
+}
+//problema2
+function twonumberslarge($arr){
+	$max=array(-100,-100);
+	for($i=0;$i<count($arr);$i++){
+		if($max[1]<$arr[$i]){
+			$max[0]=$max[1];
+			$max[1]=$arr[$i];
+			}
+		else if($max[0]<$arr[$i]){
+			$max[0]=$arr[$i];
+			}	
+	}
+	echo '<pre>';
+	print_r($max);
+}
+//problema3
+function getArrayDups($array)
+{
+  $newArray = array_unique($array);
```

Comment:
Implemtarea in c de la array_duplicate nu respecta cerintele http://lxr.php.net/xref/PHP_5_4/ext/standard/array.c#2777 foloseste o structura suplimentara de date.